### ISSUE TITLE
Symlinks in uploaded run tar prevents dearchiving 

### ISSUE THREAD

**Issue by TShapinsky** [NREL/alfalfa#357]

**Comment by TShapinsky** [comment]
It is fairly easy to tall the python `tarfile` library to turn symlinks into their target files. However when you do this e+ can no longer find supporting files for running the simulation.

**Comment by TShapinsky** [comment]
@axelstudios can you provide more context on what issue you're running into on windows when trying to dearchive runs? I wonder if there may be a different solution to just losing symlinks.

**Comment by axelstudios** [comment]
Sure - this is the error you'll see when attempting to extract the contents of a completed runs' .tar.gz archive: 
![image](https://user-images.githubusercontent.com/411466/195433054-1e09d9a4-fd92-404c-a497-02418239a80a.png)

That's because it contains serveral symlinks, which require admin rights on Windows to create (and in general you _really_ don't want to be using symlinks on Windows unless you have a great reason):
![image](https://user-images.githubusercontent.com/411466/195433489-82a0ca58-5706-439c-8455-b0c868019ea5.png)

When creating the tarballs it should be easy enough to have it archive actual files instead of symlinks: https://superuser.com/a/137391/1074025

**Comment by TShapinsky** [comment]
Yeah, I can dereference but it breaks the worker, and also when we have python venvs there are symbolic links to files which are not in the run directory.
